pkg/apiclient/workflow/workflow.proto

```diff
@@ -30,9 +30,11 @@ message WorkflowGetRequest {
 
 message WorkflowListRequest {
   string namespace = 1;
-  k8s.io.apimachinery.pkg.apis.meta.v1.ListOptions listOptions = 2;
+  string namePrefix = 2;
+  string namePattern = 3;
```

Comment:
let's add some comments to describe these. the comments should make it to the Swagger and the API docs are sorely lacking descriptions